kujira/chain.json

```diff
@@ -33,19 +33,19 @@
   },
   "codebase": {
     "git_repo": "https://github.com/Team-Kujira/core",
-    "recommended_version": "v0.8.4-mainnet",
+    "recommended_version": "v0.8.7",
     "compatible_versions": [
-      "v0.8.4-mainnet"
+      "v0.8.7"
```

Comment:
again, if this is compatible, please keep v0.8.4-mainnet in the list of compatible versions.

Reply:
In this instance we found out (with a fun case of non-determinism) that this patch was consensus breaking for kujira. v0.8.7 is the current and only compatible version. The previous version was actually v0.8.6 and will apphash.

Reply:
Thanks for explaining. In cases where the new version is consensus-breaking, please instead create a new version object at the end of the `versions` array (so we build up a history of versions) instead of just updating the latest one.